account/manager.go

```diff
@@ -1422,12 +1422,14 @@ func (m *manager) spendAccount(ctx context.Context, account *Account,
 				"accounts are not currently supported")
 		}
 
-		spendPkg, err = m.spendAccountExpiry(
-			ctx, account, packet, bestHeight,
+		spendPkg, err = m.signSpendTx(
+			ctx, account, packet, bestHeight, witnessType,
 		)
 
 	case multiSigWitness:
-		spendPkg, err = m.signSpendTx(ctx, account, packet, 0)
+		spendPkg, err = m.signSpendTx(
```

Comment:
I think this can be further simplified to just calling `signSpendTx` while passing the `witnessType` in directly.

Reply:
Also here is there any reason why we wouldn't just always set the height here?

Reply:
No, I don't think there's a reason. We should always be able to set the lock time to the current height. Fixed.